labs/lab1/chapter1.md

```diff
@@ -273,4 +268,4 @@ Confirm the registry is running.
 
 ```docker ps```
 
-In the next lab we will be pushing our work to this registry.
+In the [next lab](https://github.com/dustymabe/summit-2017-container-lab/blob/master/labs/lab2/chapter2.md) we will be pushing our work to this registry.
```

Comment:
maybe use a relative path here. in case this content gets cloned/copied to another location.